src/json-completion.ts

```diff
@@ -144,8 +169,10 @@ export class JSONCompletion {
     }
 
     // handle filtering
-    result.options = Array.from(collector.completions.values()).filter((v) =>
-      stripSurrondingQuotes(v.label).startsWith(prefix)
+    result.options = Array.from(collector.completions.values()).filter(
+      (v) =>
+        stripSurrondingQuotes(v.label).startsWith(prefix) ||
+        stripSurrondingQuotes(v.label) === prefix
```

Comment:
I noticed an issue where providing the full key causes it to be removed from the available completion options, which is annoying if you are lazy like me and want the insert text anyways. perhaps we can use a regular expression here

Reply:
reverted, as this doesn't fix the issue and slows down completions slightly